docs/source/02_Terminology.md

```diff
@@ -131,7 +132,35 @@ Since in practical terms the time is measured in discrete samples,
 the temporal scope includes the start time sample 
 but does not include the end time sample.
 
-## Time-block [*]
+### Time-block [*]
 
 A contiguous portion of the data recording during which some aspect of the experiment 
 is fixed or noted.
+
+## 2.2 Character sets and restrictions
+
+Starting with HED standard schema versions 8.3.0 and above, HED will allow UTF-8 characters in various settings.
+The types of characters referred to in this specification are:
+
+| Name | Description |
+| ---- | ----------- |
+| `ascii` | utf-8 codes 0 to 127 (single byte) |  
+| `non-ascii` | utf-8 codes greater than 128 (multi-byte) |  
```

Comment:
Are we going to narrow this down at all, or is the intent to truly allow *any* non ascii utf8 character(s)?  That's of course the easiest to implement

Reply:
So below 32, the only relevant characters I see are \t, \r, and \n.  Do we want to just remove the ascii designation and instead specify which < 32 characters are allowed manually when relevant?(as newline already does)

Reply:
Added `printable` and named the tab character. We'll specify individual characters as indicated in the value classes